modules/@apostrophecms/video-widget/ui/src/index.js

```diff
@@ -1,4 +1,5 @@
 export default () => {
+  apos.oembedCache ||= {};
```

Comment:
Cache oembed results while we're on the page. Video widget preview implemented in this way will never be perfect, but this reduces the OMG flicker factor.